src/Testcontainers.Azurite/AzuriteConfiguration.cs

```diff
@@ -0,0 +1,213 @@
+namespace Testcontainers.Azurite;
+
+/// <inheritdoc cref="ContainerConfiguration" />
+[PublicAPI]
+public sealed class AzuriteConfiguration : ContainerConfiguration
+{
+  /// <summary>
+  ///   Initializes a new instance of the <see cref="AzuriteConfiguration" /> class.
+  /// </summary>
+  /// <param name="azuriteServices">A value indicating which Azurite service is turned on.</param>
+  /// <param name="blobPort">The Azurite container blob port.</param>
```

Comment:
In general there is no need. Even if you run several Azurite containers at once it is fine. 

I know about few cases where the ports were changed and it was because of defensive programming. Developers connect to wrong container instance and system behaves unpredictable. And second use case was when Azurite was hidden behind Mockserver to simulate error responses from one specific service or some cases.

My original intention was to implement full featured module based on [documentation](https://github.com/Azure/Azurite#supported-command-line-options) for Azurite to eliminate needs to do own implementations. Basically take u module and based and set it up same way as I can do directly via docker or in command line.

I was thinking also to add `string ContainerConnectionString { get; }` or maybe `GetContainerConnectionString(string nameOrAliasOrIp)` which is value used by other containers running in same docker, network etc. And if ports will be changed then I need to know new port for each service to be able to generate any connection string properly.

As you know I asked what is Testcontainers approach because I am aware that kibd of this module will need more effort for maintenance when newer version of Ayurite will be released. If you think it is too much then I will use just standard ports as other modules do.

Reply:
> Basically take u module and based and set it up same way as I can do directly via docker or in command line.

Something I also had to learn. Looking into Java's implementation (it has mature modules), modules are not a 1:1 implementation of their image. Its kinda configuration that is immediately ready to use. Users should not need to spend a lot of time configuring modules.

E.g. what is the benefit from disabling services? When does a developer need to disable a service? If we look into the LocalStack module we do not configure any services either. The container is not faster nor slower.

Reply:
yes, we do as a quick acces to the endpoint